api/issue.go

```diff
@@ -102,7 +102,25 @@ type IssueCreate struct {
 	AssigneeID       int       `jsonapi:"attr,assigneeId"`
 	SubscriberIDList []int     `jsonapi:"attr,subscriberIdList"`
 	RollbackIssueID  *int      `jsonapi:"attr,rollbackIssueId"`
-	Payload          string    `jsonapi:"attr,payload"`
+	// CreateContext is used to create the issue pipeline and not persisted.
+	CreateContext string `jsonapi:"attr,createContext"`
+	Payload       string `jsonapi:"attr,payload"`
+}
+
+// DatabaseCreateContext is the issue create context for creating a database.
```

Comment:
Would be good to explain why we do this. To be more specific, what makes use choose to do this in the backend instead of doing this in the frontend. And is this only for create database or it would be used for other task type like update database schema as well?

Because this is not an obvious change by itself and if we don't make it clear here, other people will be confused about the line we draw between implementing it in frontend and backend.